src/scss/includes/components.scss

```diff
@@ -8,8 +10,10 @@
 
 :host([theme='light']) {
   @include componentLightThemeVars();
+  @include generate-status(light)
```

Comment:
Stencil application shouldn't use any SASS `@mixins` from `tecnologia-design-tokens`, the design tokens library will generate all files, and only that.